src/pages/_document.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import Document, { DocumentContext, DocumentInitialProps } from 'next/document'
+import { ServerStyleSheet } from 'styled-components'
+
+export default class MyDocument extends Document {
```

Comment:
Could we use a better name for this than `MyDocument`, or add a comment to indicate why a custom `Document` is necessary?

Reply:
ah, yeah.  I copied it over from the docs. Will add a comment with a link.

As for the name, can you help me with a suggestion? I'm blanking on a good name: `CustomDoc` is as bad as `MyDocument` IMO. Then, there's only one single `_document.tsx` in next js, so we cannot make it super specific like `StyledComponentsBootstrapDoc` or something like that, because if we ever change it won't make sense.

Reply:
changed to `DocumentWithStyles`. Let me know if you'd prefer something different.